### PR TITLE
Feat/bg initiator generic

### PR DESCRIPTION
**Pull request by nbittich**
related to https://github.com/lblod/delta-producer-background-jobs-initiator/pull/3

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -730,16 +664,10 @@ services:
 ################################################################################
 # DELTA SUBSIDIES: START
 ################################################################################
-  delta-producer-background-jobs-initiator-subsidies:
+  delta-producer-background-jobs-initiator-generic:
```
Comment (claire-lovisa) [inline docker-compose.yml:667]: Two minor remarks but:
1. Wouldn't it make more sense to put this service in the `# DELTA GENERAL` section ?
2. Couldn't we just omit the `-generic` in the same ? Especially if it's in the DELTA GENERAL section, in speaks for itself ?

Comment (claire-lovisa) [inline docker-compose.yml:667]: The docker-compose.dev.yml file should also be updated, to add the `restart: "no"` on the generic initiator and remove the specific job initiators

Comment (claire-lovisa) [inline docker-compose.yml:667]: The docker-compose.dev.yml file should also be updated, to add the restart: "no" on the generic initiator and remove the specific job initiators

**Review by claire-lovisa — CHANGES_REQUESTED** [review]
Without the update on the docker-compose.dev.yml, the tack doesn't start :grin: Other than that it seems to be working fine !

File: config/delta-producer/background-jobs-initiator/config.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
+[
```
[the remark this answers is not in the document]
Comment (nbittich) [reply config/delta-producer/background-jobs-initiator/config.json:1]: @claire-lovisa sorry this branch wasn't meant to be reviewed. we do it in the harvester first to test if everything's ok there.

**Comment by aatauil** [comment]
This PR is more then a year old and most likely not needed anymore. Please feel free to reopen it if I am mistaking.

**Pull request closed without merging**